lib/deimos/utils/db_poller/base.rb

```diff
@@ -65,12 +85,12 @@ def start
         # Grab the PollInfo or create if it doesn't exist.
         # @return [void]
         def retrieve_poll_info
-          @info = Deimos::PollInfo.find_by_producer(@config.producer_class) || create_poll_info
+          @info = Deimos::PollInfo.find_by_producer(@resource_class.to_s) || create_poll_info
         end
 
         # @return [Deimos::PollInfo]
         def create_poll_info
-          Deimos::PollInfo.create!(producer: @config.producer_class, last_sent: Time.new(0))
+          Deimos::PollInfo.create!(producer: @resource_class.to_s, last_sent: Time.new(0))
```

Comment:
@dorner I put this in just for now so I could test it with item-platform locally to verify it works. I wanted to get your thoughts though on if maybe we should rename the `producer` attribute to be something else that makes more sense considering it can be either a single producer class name, or the poller class name.

Reply:
Yes, I think that makes a lot of sense.

Reply:
@dorner In order to do this, would I just need to update the template files in the generator directory here:
`lib/generators/deimos/db_poller/templates/migration`
`lib/generators/deimos/db_poller/templates/rails3_migration`
And rename the producer attribute in the db migration? Or would I need make these alterations somewhere else?

Reply:
I forgot this was a DB column. Hmm... I'm not sure it's worth having to deal with changing existing applications in order to do this. Let's leave it.

Reply:
Okay sounds good.